src/dvclive/live.py

```diff
@@ -79,6 +85,7 @@ def _cleanup(self):
             shutil.rmtree(
                 Path(self.dir) / data_type.subfolder, ignore_errors=True
             )
+        shutil.rmtree(Path(self.dir) / self.PARAMS_DIR, ignore_errors=True)
```

Comment:
Why not handle whether or not to remove the dir base on `self._resume` here instead of inside the `log_param` method?

Reply:
Not sure what you mean here